src/components/App.js

```diff
@@ -1,8 +1,10 @@
 import BooksList from '../containers/BooksList';
 import BooksForm from '../containers/BooksForm';
+import CategoryFilter from './CategoryFilter';
 
 const App = () => (
   <div>
+    <CategoryFilter />
```

Comment:
- [ ] The CategoryFilter component should be imported into BookList container with the handleFilterChange function passed down to it. It shouldn't be placed in App.js.